backend/src/main/java/de/bund/digitalservice/ris/search/service/LiteratureService.java

```diff
@@ -132,6 +132,6 @@ public List<Literature> getByDocumentNumber(String documentNumber) {
    */
   public Optional<byte[]> getFileByDocumentNumber(String documentNumber)
       throws ObjectStoreServiceException {
-    return literatureBucket.get(String.format("%s.xml", documentNumber));
+    return literatureBucket.get(String.format("%s.akn.xml", documentNumber));
```

Comment:
Good catch! But can we double-check with team valid to be sure that this will always be the pattern?